src/I18n/Time.php

```diff
@@ -45,6 +45,22 @@ class Time extends Carbon implements JsonSerializable
     protected static $_toStringFormat = [IntlDateFormatter::SHORT, IntlDateFormatter::SHORT];
 
     /**
+     * The format to use when when converting this object to json
+     *
+     * The format should be either the formatting constants from IntlDateFormatter as
+     * described in (http://www.php.net/manual/en/class.intldateformatter.php) or a pattern
+     * as specified in (http://www.icu-project.org/apiref/icu4c/classSimpleDateFormat.html#details)
+     *
+     * It is possible to provide an array of 2 constants. In this case, the first position
+     * will be used for formatting the date part of the object and the second position
+     * will be used to format the time part.
+     *
+     * @var mixed
```

Comment:
You specified `string|array|int` for `$format` param of `setJsonEncodeFormat()` method which is used to set this property. So this property should have same types specified.